release/resources/module_list.json

```
@@ -23,6 +23,10 @@
             "name": "module-ballerina-data.xmldata",
             "version_key": "stdlibDataXmldataVersion"
         },
+        {
+            "name": "module-ballerina-data.csv",
+            "version_key": "stdlibDataCsvVersion"
+        },
```

Shall we add this entry before the jsondata module? need to preserve the alphabetical order
```suggestion
        {
            "name": "module-ballerina-data.csv",
            "version_key": "stdlibDataCsvVersion"
        },
```